package.json

```diff
@@ -179,5 +179,9 @@
     "postcss-calc": "6.0.2",
     "jsdom": "16.7.0",
     "postcss-modules": "^3.2.2"
+  },
+  "dependencies": {
+    "npm-cli-login": "^1.0.0",
+    "verdaccio": "^5.13.0"
```

Comment:
Ah, one minor thing. These should be added as devDependencies: https://classic.yarnpkg.com/lang/en/docs/cli/add/#toc-yarn-add-dev-d